op/main.py

```diff
@@ -148,6 +153,9 @@ def main(charm_class):
     # of events from debugging sessions.
     juju_event_name = Path(sys.argv[0]).name
 
+    if not juju_event_name:
```

Comment:
Empty line above may be dropped. These two lines are very tightly related: set and check.